addons/mail/static/src/scss/activity_view.scss

```diff
@@ -170,3 +143,7 @@
         }
     }
 }
+
+.o_activity_view_table {
+    height: 1px; // this value is ignored on the table but it is necessary to ensure h-100 works on descendants to vertically center content inside cells
```

Comment:
@alexkuhn I believe I fixed the visual bug, but the fix is not pretty

https://stackoverflow.com/a/46110096

Reply:
Works much better!